doozer/doozerlib/cli/scan_osh.py

```diff
@@ -173,19 +177,29 @@ def create_update_ocpbugs_ticket(self, packages: dict):
 
             # Check if ticket exits
             summary = f"{self.version} SAST scan issues for {brew_package_name}"
-            query = f"project={self.jira_project} AND ( summary ~ '{summary}' ) AND " \
-                    "statusCategory in ('New', 'Assigned')"
             description = "{*}{color:#FF0000}WARNING{color}{*}\n" + MESSAGE.format(package_name=brew_package_name,
                                                                                    scan_id=data["latest_coverity_scan"],
                                                                                    nvr=data["nvr"],
                                                                                    brew_build_id=data["brew_build_id"]
                                                                                    )
 
-            @retry(reraise=True, stop=stop_after_attempt(10), wait=wait_fixed(3))
-            def search_issues(query):
-                return self.jira_client.search_issues(query)
-
-            open_issues = search_issues(query)
+            # Check if this is the first time that we are raising the ticket for this component
+            query = f"project={self.jira_project} AND ( summary ~ '{summary}' ) AND " \
+                    "statusCategory not in ('New', 'Assigned')"
+            closed_issues = self.search_issues(query)
+
+            if not closed_issues:
+                # There are no previous tickets for this component, raise the ticket this time if
+                # vulnerabilities are found
+                self.skip_diff_check = True
+
+            # Find if there is already a "OPEN" ticket for this component.
+            # A ticket is said to be "OPEN" if statusCategory <= 'Assigned'.
+            # If we detect a net-new security issue, we should open a new ticket instead of updating the one
+            # that is in the process of being fixed.
```

Comment:
@thegreyd This is what I wanted to mention today. So if an engineer has started to work on the ticket, any new vulnerabilities should be raised as a new ticket. Right now we may not be listing out the new vulnerabilities found, but we could add that as a refinement.